README.md

```diff
@@ -0,0 +1,167 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone - Budget app <a name="about-project"></a>
+
+**Ruby on Rails**
+
+> budget app helps `users` to manage their budgets. It has a list of transactions associated with a category, so that you can see how much money you spent and on what. It is built on Rails using `Rubygem` as software packages system.
+
+## 🛠 Built With <a name="built-with"></a>
+
+> `Ruby 3.1.3` > `postgresql` > `Rails 7`
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Backend</summary>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Categories**
+- [ ] **Transactions**
+- [ ] **Expenditures**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Link](https://expenditor.herokuapp.com/)
+
```

Comment:
- [ ] As mentioned by the previous reviewer and fro project requirements, kindly make sure your project is deployed and accessible online. Right now, you current link is not working.